activitypub.php

```diff
@@ -270,7 +274,7 @@ function get_plugin_version() {
 		'activitypub',
 		'\Activitypub\Cli',
 		array(
-			'shortdesc' => __( 'ActivityPub related commands: Meta-Infos, Delete and soon Self-Destruct.', 'activitypub' ),
+			'shortdesc' => 'ActivityPub related commands to manage plugin functionality and the federation of posts and comments.',
```

Comment:
I don't know! See #987. WP CLI [doesn't seem to translate their output](https://github.com/wp-cli/wp-cli/blob/e214df253932712ebe0669d50bb485276a9a5102/php/commands/src/CLI_Command.php#L243). And docs pulled from function comments are also not localized.

Do you use it in German?

Reply:
The text seems to be translated...

<img width="1075" alt="Screenshot 2024-11-12 at 21 45 49" src="https://github.com/user-attachments/assets/266ea0c6-430b-4ef2-be8e-4887cd6001c3">

So the questions might be more like: do we want that? even more because it seems to be the only translated text!?!

Reply:
let's switch to the other PR!